contracts/app/examples/local_daemon.rs

```diff
@@ -50,26 +47,26 @@ fn main() -> anyhow::Result<()> {
     }
 
     // Publish the App to the Abstract Platform
-    publisher.publish_app::<AppInterface<Daemon>>()?;
+    publisher.publish_app::<MyAppInterface<Daemon>>()?;
 
     // Install the App on a new account
 
     let account = abstract_client.account_builder().build()?;
     // Installs the app on the Account
-    let app = account.install_app::<AppInterface<_>>(&AppInstantiateMsg { count: 0 }, &[])?;
-
-    // Import app's endpoint function traits for easy interactions.
-    use app::{AppExecuteMsgFns, AppQueryMsgFns};
-    assert_that!(app.count()?.count).is_equal_to(0);
-
-    // Execute the App
-    app.increment()?;
-
-    // Query the App again
-    assert_that!(app.count()?.count).is_equal_to(1);
-
-    // Note: the App is installed on a sub-account of the main account!
-    assert_ne!(account.id()?, app.account().id()?);
+    let _app = account.install_app::<MyAppInterface<_>>(&MyAppInstantiateMsg {}, &[])?;
+
+    // // Import app's endpoint function traits for easy interactions.
+    // use my_app::{MyAppExecuteMsgFns, MyAppQueryMsgFns};
+    // assert_that!(app.count()?.count).is_equal_to(0);
+    //
+    // // Execute the App
+    // app.increment()?;
+    //
+    // // Query the App again
+    // assert_that!(app.count()?.count).is_equal_to(1);
+    //
+    // // Note: the App is installed on a sub-account of the main account!
+    // assert_ne!(account.id()?, app.account().id()?);
```

Comment:
Commented out because we removed the counter example

Reply:
What's the reason for removing the counter code? It's the default example code in CosmWasm so it's pretty nice to adhere to that imo.